README.md

```
@@ -0,0 +1,79 @@
+- **packages/core**  
+  It contains Citronote's core logic, such as abstract classes and managers.
+- **packages/app**  
+  This is the front-end used in the browser and desktop versions of Citronote, built with React. It can run as a standalone web app, or as a desktop app wrapped in electron. The conditions are separated by the appEnv variable at build time.
+- **packages/ui**  
+  Contains React UI components. Separated from packages/app so that it can be exposed as a plugin in the future.
+- **packages/desktop**  
+  This is the desktop version of Citronote, which wraps packages/apps in Electron and includes an API for ipc and processing for saving to appData in the desktop version.
+- **packages/shared**  
+  Contains common utilities used by many packages.
+
+## Information about Hackatime projects
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)